src/sentry/services/http.py

```diff
@@ -27,6 +27,7 @@ def convert_options_to_env(options):
 
 class SentryHTTPServer(Service):
     name = "http"
+    protocol_options = "http", "uwsgi"
```

Comment:
No https? I think there's also `http11`.

Reply:
It's safer for us now to not try and accommodate anything else we don't actually run so we don't support it. If the time comes, we'll just add something here, but I'm going to advise against that.